libexec/util-build

```diff
@@ -332,16 +332,16 @@ __load_xcode_build_settings () {
 		s/$/'/;
 		" | \
 		grep \
-		-e "CONFIGURATION\|INFOPLIST_PATH\|PRODUCT_SETTINGS_PATH\|SRCROOT"\
```

Comment:
Let me double check the intention of original implementation ... was this to capture the line `CONFIGURATION=` only, or was this meant to capture other lines such as `CONFIGURATION_BUILD_DIR=` and `PODS_CONFIGURATION_BUILD_DIR=` ?

Reply:
Yes it was,  `relax` uses only `CONFIGURATION` environment variable, not other variables including the word 👍

Reply:
OK, and thank you!